saleor/core/models.py

```diff
@@ -119,6 +120,7 @@ class Job(models.Model):
     status = models.CharField(
         max_length=50, choices=JobStatus.CHOICES, default=JobStatus.PENDING
     )
+    pending_target = models.CharField(max_length=50, choices=PendingTarget.CHOICES)
```

Comment:
If we change something in any of the core functionality, we should do it in separate PR merged to master instead of the feature branch. 

If anybody wants to use this field, they'll have to wait for your feature branch to be merged.

Reply:
My god, that meant to be added to InvoiceJob model as it is specific. Will change that!